src/Features/Core/Portable/DocumentationComments/AbstractDocumentationCommentFormattingService.cs

```diff
@@ -527,6 +621,23 @@ private static void AppendTextFromAttribute(FormatterState state, XElement eleme
                 new SymbolDisplayPart(kind, symbol: null, text: TrimCrefPrefix(crefValue)));
         }
 
+        internal static IEnumerable<SymbolDisplayPart> TypeParameterRefToSymbolDisplayParts(
+            string crefValue, ISymbol typeResolutionSymbol, int position, SemanticModel semanticModel, SymbolDisplayFormat format = null)
+        {
+            if (semanticModel != null)
+            {
+                var typeParameterIndex = typeResolutionSymbol.OriginalDefinition.GetAllTypeParameters().IndexOf(tp => tp.Name == crefValue);
+                if (typeParameterIndex > -1)
+                {
+                    return typeResolutionSymbol.GetAllTypeArguments()[typeParameterIndex].ToMinimalDisplayParts(semanticModel, position, format);
+                }
+            }
+
+            // if any of that fails fall back to just displaying the raw text
+            return SpecializedCollections.SingletonEnumerable(
+                new SymbolDisplayPart(SymbolDisplayPartKind.TypeParameterName, symbol: null, text: TrimCrefPrefix(crefValue)));
```

Comment:
can youjust fall back to `CrefToSymbolDisplayParts` since htat what was called originally?

Reply:
i'd feel bad to do that, because it's not a cref

Reply:
CrefToSymbolDisplayParts that used to be called for these values.  so this allows us to use the existing codepath for typeparams when they don't work with your new logic.